### ISSUE TITLE
Publishing media

### ISSUE THREAD

**Issue by innodan** [solewniczak/dokuwiki-plugin-approve#3]
Is there a possibility to use this plugin for media, too?

**Comment by solewniczak** [comment]
No, it isn't possible in this version but it's technically possible and 
should not be hard. I will think about that when I update the plugin.

W dniu 03.08.2017 o 12:27, innodan pisze:

**Comment by innodan** [comment]
Hey, it would be really great if you can realize that function within the next update. 👍

**Comment by aov-arnold** [comment]
@solewniczak Is there any chance that this feature will be added? Media should be included in the approval and changed media should lead to a new draft.
